src/app/board-tile/board-tile.component.html

```diff
@@ -1 +1 @@
-<div class="tile" [id]="tileIdx">{{letter}}</div>
+<div [class]="['tile', gameplayService.evaluateLetter$(rowIdx, tileIdx) | async]" [id]="tileIdx">{{letter}}</div>
```

Comment:
This is something Andrew brought up that I think (hope!) we've all been handling ok just by following patterns in our app...but if you call a function inside a template, Angular change detection will run that code (call that function) many many times. Try putting a console.log inside `evaluateLetter$()` to see. In this case I think what you want is to call `evaluateLetter` whenever one or both of rowIdx and tileIdx get changed, and then use the most recent return from `evaluateLetter` in the template.

Reply:
I am seeing a ton of `console.log`'s... I think what I want is to call evaluateLetter when the component is initialized, and then I expect to update that letter tile only once. Something else that's challenging me right now is that the tests are failing--e.g. after `acceptedGuesses$` is updated `compiled.querySelector('.incorrect' | '.correct' | '.in-word')?.textContent` is returning `undefined` because the class doesn't exist, when it should be returning `letter`--but when I run the dev server in localhost it seems to be updating the classes just fine. I'm going to continue playing around with it to see if I can solve this problem!